R/dwdData.R

```diff
@@ -0,0 +1,204 @@
+################################################################################
```

Comment:
I personally don't like the "dwd" as a shorthand for "download". I also think that having `dwdData` and `downloadData` is confusing, so this function should be renamed entirely. How about using "get" or "fetch" prefix, with "url" or similar as suffix (*e.g.*, `getURL` -- but I think this is used in another package)?